test/plugin/base_test.rb

```diff
@@ -1608,22 +1649,32 @@ def verify_json_log_entries(n, params, payload_type = 'textPayload')
         labels ||= request['labels']
         labels.merge!(entry['labels'] || {})
 
-        assert_equal \
-          "projects/#{params[:project_id]}/logs/#{params[:log_name]}", log_name
+        expected_log_name = \
+          if params[:log_name]
+            "projects/#{params[:project_id]}/logs/#{params[:log_name]}"
+          else
+            # For a request with multiple tags, each entry's log name should be
```

Comment:
I would prefer not to build in this assumption, but if you choose to do so, please document at the top of `verify_log_entries` (because otherwise it's hard to dig for).
I still think it makes sense to do this check separately in the test case for anything that isn't a constant log name.

Reply:
I see. Moved this check back to the individual test case.